README.md

```diff
@@ -16,7 +16,11 @@ remove deployment platforms, or test with a different suite.
 ## Features
 * Python-centric skeletal structure with initial module files
 * Pre-configured `setup.py` for installation and packaging
+<<<<<<< HEAD
```

Comment:
Looks like there are some incompletely merged files left in here?

Reply:
Indeed it does, thanks! This was so out of date I had a hard time with the rebase :) I can't figure out what's going on with my appveyor build either 🤡